scripts/dbgen/main.go

```diff
@@ -459,8 +459,7 @@ func orderAndStubDatabaseFunctions(filePath, receiver, structName string, stub f
 		return xerrors.Errorf("format package: %w", err)
 	}
 	data, err := imports.Process(filePath, buf.Bytes(), &imports.Options{
-		Comments:   true,
-		FormatOnly: true,
+		Comments: true,
```

Comment:
Expected change?

Reply:
yes, it's expected. The generation script was not importing the sql package, and allowing it to process imports fixed that.